test/integration/targets/mysql_variables/tasks/mysql_variables.yml

```diff
@@ -308,3 +308,53 @@
     login_unix_socket: '{{ mysql_socket }}'
     login_user: root
     login_password: '{{ root_pass }}'
+
+#=========================================
+# Check mode 'persist' and 'persist_only':
+#
+- name: set mysql variable to a temp value
+  mysql_variables:
+    variable: '{{set_name}}'
+    value: '200'
+    login_unix_socket: '{{ mysql_socket }}'
+    login_user: root
+    login_password: '{{ root_pass }}'
+
+- name: update mysql variable value (expect changed=true) in persist mode
```

Comment:
How about idempotence checks?

Reply:
yep, it started to work (after your question)... :) thanks! Good question is often better than a PR itself...